cluster/addons/calico-policy-controller/calico-node-daemonset.yaml

```diff
@@ -45,14 +45,22 @@ spec:
               value: "calico-typha"
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
+            - name: FELIX_HEALTHENABLED
```

Comment:
This is a duplicate of one already set below.

Reply:
Removed.